csaf_2.1/prose/edit/src/conformance.md

```diff
@@ -127,9 +127,14 @@ Secondly, the program fulfills the following for all items of:
 * `/product_tree/relationships[]`: If more than one `prod:FullProductName` instance is given,
   the CVRF CSAF converter converts the first one into the `full_product_name`.
   In addition, the converter outputs a warning that information might be lost during conversion of product relationships.
-* `/vulnerabilities[]/cwe`: If more than one `vuln:CWE` instance is given,
-  the CVRF CSAF converter converts the first one into `cwe`.
-  In addition, the converter outputs a warning that information might be lost during conversion of the CWE.
+* `/vulnerabilities[]/cwes[]`:
+  * The CVRF CSAF converter MUST determine the CWE specification version the given CWE was selected from by
+    using the latest version that matches the `id` and `name` exactly and was published prior to the value of
+    `/document/tracking/current_release_date` of the source document. If no such version exist, the first matching version
```

Comment:
Nit: Suggest to start Sentences on new lines.

Reply:
Done in 6f5b64a1